.github/workflows/dapr-3rdparty-container.yaml

```diff
@@ -0,0 +1,41 @@
+name: dapr-mirror-containers
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' # trigger on the 1st of every month at midnight
+  workflow_dispatch:
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      - # ghcr logins for pushing image after testing
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_REGISTRY_ID }}
+          password: ${{ secrets.DOCKER_REGISTRY_PASS }}
+      - # copy 3rd party zookeeper image from dockerhub to ghcr
+        name: Push Image (zookeeper) to target registries
+        uses: akhilerm/tag-push-action@v2.0.0
+        with:
+          src: wurstmeister/zookeeper:latest
+          dst: |
+            ghcr.io/${{ secrets.DOCKER_REGISTRY_ID }}/3rdparty/zookeeper:latest
```

Comment:
Do we need to publish to Docker Hub?

Reply:
In this case, the mistake is that you are using the DOCKER_REGISTRY_ID instead of our GitHub org. Please, change this to ${{ github.repository_owner }}